api4/user.go

```diff
@@ -1483,6 +1484,25 @@ func revokeAllSessionsForUser(c *Context, w http.ResponseWriter, r *http.Request
 	ReturnStatusOK(w)
 }
 
+func revokeAllSessionsAllUser(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
I did think about naming it like that, but thought that it would be repetitive for people, so I went with just one all that would carry the meaning. But this is fine with me!